internal/handlers/common/params.go

```diff
@@ -100,6 +100,26 @@ func AssertType[T types.Type](value any) (T, error) {
 	return res, nil
 }
 
+// GetLimitParam returns limit value from provided query document.
+func GetLimitParam(doc *types.Document) (int64, error) {
+	v, _ := doc.Get("limit")
+	if v == nil {
+		return 0, nil
+	}
+
+	res, err := commonparams.GetWholeNumberParam(v)
+	if err != nil {
+		msg := fmt.Sprintf(
+			`BSON field '%s' is the wrong type '%s', expected type '%s'`,
+			"limit", commonparams.AliasFromType(v), commonparams.AliasFromType(res),
+		)
+
+		return res, commonerrors.NewCommandErrorMsgWithArgument(commonerrors.ErrTypeMismatch, msg, "limit")
+	}
+
+	return res, nil
+}
+
```

Comment:
Helper needed to fix a bug (the msg_explain didn't take int values, but only longs, when msg_find and mongodb accepted all whole values).

Reply:
Could you add a test for it please? :hugs:

Reply:
This comment isn't addressed.

Reply:
Could you add a test for int please. New logic introduction without test is not clear.

Also does this need to be a common function at all? The thing is `GetLimitParam` is not accurate description of this function as far as I understand, it cannot be used for getting limit param in find. :hugs:

Reply:
> Also does this need to be a common function at all? The thing is GetLimitParam is not accurate description of this function as far as I understand, it cannot be used for getting limit param in find. 🤗

It can be used in find, we just don't use it there.